api/api.yaml

```diff
@@ -215,16 +215,27 @@ paths:
           $ref: '../result/request.yaml#/components/responses/InternalServerError'
 
   /results/state:
+    description: >-
+      State API
     post:
-      tags: ['Metrics']
-      operationId: get_state_metrics
+      tags: ['State']
+      operationId: get_states
+      requestBody:
+        description: >-
+          Request to traffic generator for states of choice
```

Comment:
"for states of selected configuration items" maybe ?

Reply:
The first level selection is choice, so there is a clear link between argument and description. Also this description i s matching with GetMetrics() function